src/components/utils/split-button.js

```diff
@@ -241,6 +241,9 @@ const SplitButton = ({
                 anchorEl={anchorRef.current}
                 role={undefined}
                 transition
+                style={{
+                    zIndex: 2,
```

Comment:
you can make it 5 for example to solve the problem above

and you can move it to the useStyles to make the migration to emotion easier.